src/BotPlutusInterface/Contract.hs

```diff
@@ -299,8 +299,8 @@ writeBalancedTx contractEnv (Right tx) = do
             , cmdOutParser = const ()
             }
 
-    collectBudgetStats txId txPath = do
-      let path = Text.unpack txPath
+    collectBudgetStats txId pabConf = do
+      let path = Text.unpack (Files.txFilePath pabConf "signed" (Tx.txId tx))
       b <- firstEitherT (Text.pack . show) $ newEitherT $ estimateBudget @w (Signed path)
```

Comment:
I actually need to make error message here more concrete, so it could be distinguished from budget estimation error during balancing.

Reply:
Done